sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.15.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.14.3 (2024-05-01)
 
 ### Bugs Fixed
 
+- Fixed a bug where mismatch Netty versions were always being reported, even if they were correct. ([#39591](https://github.com/Azure/azure-sdk-for-java/pull/39591))
+
 ### Other Changes
 
+- If a Reactor Netty `HttpClient` is passed and has a `LoggingHandler` configured the 
+  `NettyAsyncHttpClientBuilder.wiretap` value is ignored, the builder method is now deprecated as well. ([#39976](https://github.com/Azure/azure-sdk-for-java/pull/39976))
```

Comment:
Deprecating an API should require a minor version bump.

https://semver.org/#spec-item-7

> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backward compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.